src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Telemetry/TelemetryScope.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Razor.PooledObjects;
+
+namespace Microsoft.AspNetCore.Razor.Telemetry;
+
+internal sealed class TelemetryScope : IDisposable
+{
+    public static readonly TelemetryScope Null = new();
```

Comment:
```suggestion
    public static readonly TelemetryScope None = new();
```

Or `Empty`

Reply:
`None` seems better than `Empty` to me. However, I do feel that `Null` is the clearer and it's not uncommon in .NET APIs used for similar purposes:

* https://learn.microsoft.com/en-us/dotnet/api/system.io.streamreader.null?view=net-7.0
* https://learn.microsoft.com/en-us/dotnet/api/system.io.binarywriter.null?view=net-7.0
* https://learn.microsoft.com/en-us/dotnet/api/system.io.stream.null?view=net-7.0#system-io-stream-null

Happy to change it if you feel strongly though.

Reply:
I didn't know we already had `Null` commonly used like that! Works for me